tests/test_fsclient.py

```diff
@@ -119,9 +125,19 @@ def test_all_documents():
     pass
 
 
-@pytest.mark.skip("Not written")
 def test_insert_one():
-    pass
+    client = FileSystemClient(rc)
+
+    # with TempDirectory() as di:
+    #     temp_dir = Path(di.path)
+    #     cif_bitstream = bytearray(cm[0], 'utf8')
+    #     d.write(f"test_cif.cif",
+    #             cif_bitstream)
+    #     test_cif_path = temp_dir / f"test_cif.cif"
+
+    client.insert_one("local", "calculated", test_cif_json)
```

Comment:
we don't seem to be testing much here, except that the code doesn't crash.   Normally I would want to test that the behavior is right.  What behavior do we want `insert_one` to have?  I could imagine different behaviors for:

1. gets a valid db that is connected
2. invalid db
3. gets a valid thing to insert
4. invalid thing to insert

and so on.